firebase-appdistribution/firebase-appdistribution.gradle

```diff
@@ -18,7 +18,9 @@ plugins {
 }
 
 firebaseLibrary {
+    libraryGroup "appdistribution"
     publishJavadoc = false
+    publishReleaseNotes = true
```

Comment:
we don't add this to other `true` sdks, remove?

Reply:
Good catch, fixed.